cypress/integration/filter_reviews.spec.js

```diff
@@ -82,21 +82,22 @@ context('Filter reviews', () => {
         .type(data.lessons)
         .should('have.value', data.lessons)
       cy.get('button').contains('Submit').click()
+      cy.get('[data-cy=homebutton]').click()
     })
     cy.visit('/reviews')
     cy.get('button').contains('Show all').click()
     mockData.forEach((data) => {
       cy.get('h2').contains(data.title)
     })
-    cy.get('button').contains('Book').click()
+    cy.get('[data-cy=filterbar]').find('button').contains('Book').click()
```

Comment:
If the button is the only thing with the text 'Book" you can try writing  `cy.contains('Book').click()`

Reply:
It is not, I have the tag book on my reviews. Needed to add the data-cy because my tests failed.